### PR TITLE
Add a Clockwork Orange book

### PR DESCRIPTION
**Pull request by Alex-Keyes**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -526,6 +526,14 @@ My favorite of Murakami's. Great mix of quirky, mundane, and fascinating ideas.
 
 _Dystopian novels deal with imaginary communities or societies that are undesirable or frightening._
 
+#### [A Clockwork Orangee](https://www.goodreads.com/book/show/227463.A_Clockwork_Orange) (1962) *by [Anthony Burgess](https://en.wikipedia.org/wiki/Anthony_Burgess)
```
Comment (RichardLitt) [inline readme.md]: Needs a closing asterix.

File: readme.md [hunk 2]
```diff
@@ -526,6 +526,14 @@ My favorite of Murakami's. Great mix of quirky, mundane, and fascinating ideas.
...
+[3.99] 
+
+Better than the movie imho.  Written in a slang language called Nadsat, the book really draws you into the world Alex occupies, as opposed to Kubrick's version.  The endings are also different! - [@alexkeyes](https://github.com/alex-keyes)
```
Comment (RichardLitt) [inline readme.md]: Do we have Kubrick's version? Can we link to it? 

Might also want to say that this is the original.

Comment (Alex-Keyes) [reply readme.md]: The movie is mentioned on the list, I improved the wording of the sentence, so it's clear I'm talking about the movie.

File: readme.md [hunk 3]
```diff
@@ -526,6 +526,14 @@ My favorite of Murakami's. Great mix of quirky, mundane, and fascinating ideas.
...
 _Dystopian novels deal with imaginary communities or societies that are undesirable or frightening._
 
+#### [A Clockwork Orangee](https://www.goodreads.com/book/show/227463.A_Clockwork_Orange) (1962) *by [Anthony Burgess](https://en.wikipedia.org/wiki/Anthony_Burgess)*
+[3.99] 
+
+Better than the movie imho.  Written in a slang language called Nadsat, the book really draws you into the world Alex occupies, as opposed to Kubrick's version of the story, portrayed in the movie.  The endings are also different! - [@alexkeyes](https://github.com/alex-keyes)
+
+>A vicious fifteen-year-old "droog" is the central character of this 1963 classic, whose stark terror was captured in Stanley Kubrick's magnificent film of the same title.
+>
+>In Anthony Burgess's nightmare vision of the future, where criminals take over after dark, the story is told by the central character, Alex, who talks in a brutal invented slang that brilliantly renders his and his friends' social pathology. A Clockwork Orange is a frightening fable about good and evil, and the meaning of human freedom. When the state undertakes to reform Alex—to "redeem
```
Comment (RichardLitt) [inline readme.md]: Seems to cut off, and needs an empty line before next entry.

**Comment by RichardLitt** [comment]
Merged in 69ba1dfc598f72eab02fa392efc851eeadc0cc53. Thanks, @Alex-Keyes!

**Pull request closed without merging**